iml-api/src/graphql/mod.rs

```diff
@@ -1139,6 +1027,32 @@ impl MutationRoot {
         transaction.commit().await?;
         Ok(true)
     }
+
+    #[graphql(arguments(
+        filesystem(description = "The filesystem to remove snapshot policies for"),
+        id(description = "Id of the policy to remove"),
+    ))]
+    /// Removes the automatic snapshot policy.
+    async fn remove_snapshot_policy(
+        context: &Context,
+        filesystem: Option<String>,
+        id: Option<i32>,
+    ) -> juniper::FieldResult<bool> {
+        sqlx::query!(
+            r#"
+                DELETE FROM snapshot_policy
+                WHERE (filesystem IS NOT DISTINCT FROM $1)
```

Comment:
Why not `!=`

Reply:
Because it any can be NULL.

Reply:
Given we can only have one policy per fs, why do we need the id at all?

Why not just make `filesystem` the only required param as it's already a unique key?

Reply:
This was written first when I thought `id` would be used in general and `fs` only for convenience.  I will update.